### PR TITLE
30 fix googleservice

### LINKED ISSUE #30: fix googleService
sending the mail doesn't work, because of "cliendId" need to replace it with "clientId

### PR DESCRIPTION
**Pull request by STom6**
# New feature
move login and register functions to the firebase functions folder.
Rename folder FirebaseDB.js to Firebasefunctions.js

# Bug fixed
replace cliendID to clientId

### CODE REVIEW

**Review by 31Nathan — DISMISSED** [review]
LGTM

**Review by JohanCDev — DISMISSED** [review]
lgtm

File: Server/Services/googleService.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const nodemailer = require('nodemailer')
 const  { google } = require('googleapis')
 const OAuth2 = google.auth.OAuth2
-const dbRealTime = require('./../RealTimeDB')
+const dbRealTime = require('../firebaseFunctions')
```
Comment (CedricCORGE) [inline Server/Services/googleService.js]: Change var name to firebaseFunctions

**Review by CedricCORGE — CHANGES_REQUESTED** [review]

**Review by CedricCORGE — APPROVED** [review]
lgtm

**Review by 31Nathan — APPROVED** [review]
LGTM

**Pull request merged**